packages/amplication-git-utils/src/types.ts

```diff
@@ -152,3 +152,8 @@ export interface CreateBranchArgs {
 export interface Commit {
   sha: string;
 }
+
+export interface AmplicationBotData {
```

Comment:
change the name to something along the lines of GitHubUser
```suggestion
export interface GitHubUser {
```